src/Filter/StringFilter.php

```diff
@@ -24,6 +24,16 @@ class StringFilter extends Filter
         ChoiceType::TYPE_EQUAL => '=',
     ];
 
+    /**
+     * @var bool
+     */
+    private $defaultCaseSensitive;
+
+    public function __construct(bool $defaultCaseSensitive = true)
```

Comment:
If someone sets it it’s not default anymore.
I would go without the default prefix

Reply:
It's just to keep BC for users who extend this class